src/components/overview/tasksBox/index.tsx

```diff
@@ -0,0 +1,132 @@
+import { ReactComponent as AddIcon } from '../../../assets/add.svg';
+import CheckCircleIcon from '@mui/icons-material/CheckCircle';
+import CustomSelect from '../select';
+import RadioButtonUncheckedIcon from '@mui/icons-material/RadioButtonUnchecked';
+import React from 'react';
+import { RootState } from '../../../store';
+import Tag from '../tags';
+import TasksModal from '../modals/tasks';
+import { Checkbox, FormControlLabel } from '@mui/material';
+import {
+  CheckboxsContainer,
+  Container,
+  CreateButton,
+  CreateText,
+  HeadingContainer,
+  HeadingText,
+  InputContainer,
+  StatusName,
+  StyledHr,
+  StyledInput,
+  TicketsContainer,
+  TimelineText,
+  ViewButton,
+} from './styled';
+import { ITask, tasks } from './helper';
+import {
+  setAddingTask,
+  setModalOpen,
+  setNewLabel,
+  setSelectedStatus,
+} from '../../../store/tasks/slice';
+import { useDispatch, useSelector } from 'react-redux';
+
+const TasksInfoBox = () => {
+  const {
+    label: newTask,
+    status: selectedStatus,
+    modal: modalOpen,
+    addingTask,
+  } = useSelector((state: RootState) => state.task);
+
+  const dispatch = useDispatch();
+
+  const toggleModal = () => {
+    dispatch(setModalOpen(!modalOpen));
+  };
+  const handleAddTask = () => {
+    dispatch(setAddingTask(true));
+  };
+
+  const handleCreateTask = () => {
+    if (newTask.trim() !== '') {
+      tasks.push({ label: newTask, tag: selectedStatus });
+      dispatch(setNewLabel(''));
+      dispatch(setSelectedStatus('Default'));
+      dispatch(setAddingTask(false));
+    }
+  };
+
+  const statusOptions = {
+    Urgent: 'Urgent',
+    New: 'New',
+    Default: 'Default',
+  };
+
+  const lastTasksNumber: number = -3;
+  const visibleTasks: ITask[] = tasks.slice(lastTasksNumber);
+
+  return (
+    <Container>
+      <HeadingContainer>
+        <HeadingText>Tasks</HeadingText>
+        <ViewButton onClick={toggleModal}>View all</ViewButton>
+      </HeadingContainer>
+      <TimelineText>Today</TimelineText>
+
+      {!addingTask ? (
+        <TicketsContainer>
+          <CreateText>Create new task</CreateText>
+          <AddIcon onClick={handleAddTask} />
+        </TicketsContainer>
+      ) : (
+        <InputContainer>
```

Comment:
In case if we use RTK for tasks handling, so, we can put it to form and move to the separate component

Reply:
fix: Form made with ReactHookForm asa a separate component

Additionally from Skype comments:
fixes:
Dropdown width and space between Select & Buttons.
Form made similar to SignIn, required included.
Form logic: last created showed at the top, default 'done' check removed.
Navigation items fixed.
Space at the bottom added for both Boxes.
Hr lines color fixed.